internal/consensus/reactor.go

```diff
@@ -373,9 +368,8 @@ func (conR *Reactor) Receive(e p2p.Envelope) {
 		}
 		switch msg := msg.(type) {
 		case *VoteSetBitsMessage:
-			conR.rsMtx.RLock()
-			height, votes := conR.rs.Height, conR.rs.Votes
-			conR.rsMtx.RUnlock()
+			rs := conR.getRoundState()
```

Comment:
The same rationale of the line 287 comment applies here.

We probably should get a copy of the actual consensus state.